google-datacatalog-sisense-connector/src/google/datacatalog_connectors/sisense/prepare/sisense_connector_strings_helper.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python
+#
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+import six
+import unicodedata
+
+from google.datacatalog_connectors.commons import prepare
+
+
+class SisenseConnectorStringsHelper:
+    """
+    Temporary class providing the logic to format column names currently
+    required by the Entry and Tag factories.
+
+    TODO
```

Comment:
Turn this TODO into a cleanup issue in the commons repo?

Reply:
Sure! Would you like to take this task? 😃

Reply:
can't atm :) my suggestion was more to track that.

Reply:
No worries. I've just filed a feature request: https://github.com/GoogleCloudPlatform/datacatalog-connectors/issues/54.